client/src/legacy/entwine/SubsitesTreeDropdownField.js

```diff
@@ -0,0 +1,33 @@
+/* global jQuery */
+jQuery.entwine('ss', ($) => {
+  /**
+   * Choose a subsite from which to select pages.
+   * Needs to clear tree dropdowns in case selection is changed.
+   */
+  $('select.subsitestreedropdownfield-chooser').entwine({
+    onchange() {
+      // TODO Data binding between two fields
```

Comment:
Please do something with this TODO, either remove it (preferable) or add some context to the comment e.g. linking it to something relevant so that future developers know what it is or when it can be safely removed